templates/config/nginx/php-fpm/php.ini

```diff
@@ -0,0 +1,7 @@
+# Custom PHP settings
+
+upload_max_filesize = 100M
+post_max_size = 100M
+
+[mail function]
+sendmail_path = /usr/sbin/sendmail -S mail:1025
```

Comment:
@mbtamuli does this work in docker?

Reply:
This line is for Docker only. These PRs are not related to non-docker setup.

Reply:
So did we test within docker if there is a sendmail program at `/usr/bin/sendmail`?

Reply:
```
sendmail_path = /usr/sbin/sendmail -S mail:1025
```
https://github.com/EasyEngine/dockerfiles/blob/master/php/Dockerfile#L15
I think this package `ssmtp` is installing `/usr/sbin/sendmail` because the above line is working. It is sending mail which is recieved by MailHog. 

Postfix mail is being tracked on in a different issue - https://github.com/EasyEngine/easyengine/issues/1028

Reply:
Fine. Looks good. 👍